web/src/rendered_markdown.ts

```diff
@@ -158,17 +164,21 @@ export const update_elements = ($content) => {
         }
     });
 
-    $content.find(".topic-mention").each(function () {
+    $content.find(".topic-mention").each(function (): void {
         const message = get_message_for_message_content($content);
 
         if (message && message.topic_wildcard_mentioned) {
             $(this).addClass("user-mention-me");
         }
     });
 
-    $content.find(".user-group-mention").each(function () {
+    $content.find(".user-group-mention").each(function (): void {
         const user_group_id = get_user_group_id_for_mention_button(this);
         let user_group;
+        if (user_group_id === undefined) {
+            blueslip.info("Rendered undefined user group");
+            return;
+        }
```

Comment:
Explicitly throwing an error because the use of this variable was in a try/catch (which seems like an antipattern in `get_user_group_from_id` but I didn't get into that)

Reply:
As noted in https://github.com/zulip/zulip/pull/28418/commits/8cd7a3d57fde78c247afaf112cc6517d0dc86552#r1443309261, I expect we don't need this case.

Reply:
Removed this as part of the commit I added asserting the user group is defined